src/api/app/controllers/status/required_checks_controller.rb

```diff
@@ -22,11 +22,12 @@ def index
     @required_checks = @checkable.required_checks
   end
 
-  # POST /status_reports/projects/:project_name/required_checks/:name
-  # POST /status_reports/repositories/:project_name/:repository_name/required_checks/:name
+  # POST /status_reports/projects/:project_name/required_checks
+  # POST /status_reports/repositories/:project_name/:repository_name/required_checks
   def create
     authorize @checkable
-    @required_checks = @checkable.required_checks |= names
```

Comment:
I'm still a bit puzzled how the tests ever passed this line

```
irb(main):001:0> [] |= ['one', 'two']
Traceback (most recent call last):
        1: from /home/hvogel/.orr/bin/irb:11:in `<main>'
SyntaxError ((irb):1: syntax error, unexpected tOP_ASGN, expecting end-of-input
[] |= ['one', 'two']
   ^~)
irb(main):002:0> ['one'] |= ['one', 'two']
Traceback (most recent call last):
        1: from /home/hvogel/.orr/bin/irb:11:in `<main>'
SyntaxError ((irb):2: syntax error, unexpected tOP_ASGN, expecting end-of-input
['one'] |= ['one', 'two']
        ^~)
```

Reply:
You need to assign it to a variable:

```
tmp = []
tmp |= ["one"]
# => ["one"]
```